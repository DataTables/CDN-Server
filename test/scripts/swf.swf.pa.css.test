swf/swf-0.0.1/pa-1.0.0/test.css
200
