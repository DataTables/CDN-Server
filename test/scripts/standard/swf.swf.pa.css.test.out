extensionsURL >swf/swf-0.0.1/pa-1.0.0<
extensionsList >StyleWithFiles 0.0.1, ProductA 1.0.0<
source >"swf/swf-0.0.1/pa-1.0.0"<
testString >test string<
./StyleWithFiles-0.0.1/css/style.css


./StyleWithFiles-0.0.1/css/style.styleWithFiles.css


./ProductA-1.0.0/css/producta.css


